src/roster/views.py

```diff
@@ -1504,5 +1533,7 @@ def get_context_data(self, **kwargs):
         context['current_student_name'] = self.clusive_student.user.first_name if self.clusive_student else "No student"
         context['teacher'] = self.clusive_user
         context['roster'] = self.roster
-        context['data'] = { 'days': 7 }
+        context['data'] = { 'days': self.days }
```

Comment:
My approach was to follow the Dashboard, putting all of the panel information in `context['data']`.  One change to the Dashboard's approach was to call the python dictionary `self.panel_data` instead of `self.data`, mostly because I kept forgetting what `self.data` was for.

When I merged the time scale selection code into my student reactions branch, I made a change:
```
self.panel_data['days'] = self.days
...
self.panel_data['affect'] = { ... }
...
context['data'] = self.panel_data
```
There are two ways forward.  One is to put all of the panel data into `context['panel_data']` and get rid of `context['data']`, or put all of the panel data into `context[data]`, and get rid of `context[panel_data]`.

I lean towards the first solution (`context[panel_data]`), since it's clearer what the data is for.  However, it might not be possible since some of the included templates use `data` variables, and they are shared with other parts of the system.

It's likely easier/safer to switch to putting everything into `context['data']` and use `{{ data }}` in your templates.

Reply:
Good point, @klown. I implemented the suggestion of using `context['panel_data']` for holding `self.days`.

Reply:
LGTM 👍